src/TransformerGo.cs

```diff
@@ -21,11 +21,11 @@ public override CodeModelGo TransformCodeModel(CodeModel cm)
 
             SwaggerExtensions.ProcessGlobalParameters(cmg);
             TransformEnumTypes(cmg);
+            TransformMethods(cmg);

```

Comment:
I'm not surprised this broke stuff.  The ordering here is really touchy as too many steps are interdependent.  Can you please put this back and add a comment that basically says "DON'T EVER CHANGE THIS"

Reply:
Yeah... I have to revert it and find some other way to fix this...